index.html

```diff
@@ -0,0 +1,124 @@
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+</div>
+<div class="row">
+<div class="square"><span>4</span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+  </div>
+<div class="row">
+<div class="square"><span>5</span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+</div>
+<div class="row">
+<div class="square"><span>6</span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+</div>
+<div class="row">
+<div class="square"><span>7</span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+</div>
+<div class="row">
+<div class="square"><span>8</span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+<div class="square"><span></span></div>
+</div>
+</div>
+            
+        
+        
+    
+        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+        <script>window.jQuery || document.write('<script src="js/vendor/jquery-1.10.2.min.js"><\/script>')</script>
+        <script src="js/plugins.js"></script>
+        <script src="js/main.js"></script>
+
+        <!-- Google Analytics: change UA-XXXXX-X to be your site's ID. -->
+        <script>
+            (function(b,o,i,l,e,r){b.GoogleAnalyticsObject=l;b[l]||(b[l]=
+            function(){(b[l].q=b[l].q||[]).push(arguments)});b[l].l=+new Date;
+            e=o.createElement(i);r=o.getElementsByTagName(i)[0];
+            e.src='//www.google-analytics.com/analytics.js';
+            r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
+            ga('create','UA-XXXXX-X');ga('send','pageview');
+        </script>
+    </body>
+</html>
```

Comment:
So, I think this was supposed to replace `pub/index.html`... but you probably already know that by now.